install.sh

```diff
@@ -1,46 +1,26 @@
 #!/bin/sh
-PATH=/Users/lehoff/.nix-profile/bin:/nix/var/nix/profiles/default/bin:$PATH
 
-ln -sf ${PWD}/nix-channels ~/.nix-channels
+mkdir -p ~/.nixpkgs
+
+# Configure the channels
 
-#from https://github.com/NixOS/nixpkgs/issues/149791#issuecomment-994964166
-#fixes file 'nixpkgs' was not found in the Nix search path
-export NIX_PATH=${NIX_PATH:+$NIX_PATH:}$HOME/.nix-defexpr/channels:/nix/var/nix/profiles/per-user/root/channels
+if ! grep -q nix-darwin ~/.nix-channels; then
+  echo "https://github.com/LnL7/nix-darwin/archive/master.tar.gz darwin" >> ~/.nix-channels
+fi
 
 export NIX_PATH=darwin=$HOME/.nix-defexpr/channels/darwin:$NIX_PATH
-export NIX_PATH=home-manager=$HOME/.nix-defexpr/channels/home-manager:$NIX_PATH
-export NIX_PATH=darwin-config=$HOME/.nixpkgs/darwin-configuration.nix:$HOME/.nix-defexpr/channels:$NIX_PATH
+export NIX_PATH=darwin-config=$HOME/Repos/dot-files/darwin-configuration.nix:$HOME/.nix-defexpr/channels:$NIX_PATH
```

Comment:
REPLACE THIS WITH THE PATH TO YOUR `darwin-configuration.nix` 🙏 
```suggestion
export NIX_PATH=darwin-config=$HOME/PATH_TO_YOUR_NIX_DOTFILES/darwin-configuration.nix:$HOME/.nix-defexpr/channels:$NIX_PATH
```